File: MyLanguageWeb/src/main/resources/static/js/classes/dto/other/response/response_message_response_dto.js
export class ResponseMessageResponseDTO {
    #id;
    #message;

    constructor(json) {
        if (json) {
            this.#id = json["id"];
            this.#message = json["message"];
        }
    }

    getId() {
        return this.#id;
    }

    getMessage() {
        return this.#message;
    }
}

File: MyLanguageWeb/src/main/java/ru/herooo/mylanguageweb/dto/other/response/ResponseMessageResponseDTO.java
package ru.herooo.mylanguageweb.dto.other.response;

import com.fasterxml.jackson.annotation.JsonProperty;

public class ResponseMessageResponseDTO {
    @JsonProperty("id")
    private int id;

    @JsonProperty("message")
    private String message;

    public ResponseMessageResponseDTO() {}
    public ResponseMessageResponseDTO(int id, String message) {
        this.id = id;
        this.message = message;
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getMessage() {
        return message;
    }

    public void setMessage(String message) {
        this.message = message;
    }
}
